conans/client/api/conan_api.py

```diff
@@ -0,0 +1,193 @@
+import logging
+import os
+import sys
+import time
+from io import StringIO
+
+from conans import __version__ as client_version
+from conans.client.cache.cache import ClientCache
+from conans.client.graph.graph_binaries import GraphBinariesAnalyzer
+from conans.client.graph.graph_manager import GraphManager
+from conans.client.graph.proxy import ConanProxy
+from conans.client.graph.python_requires import ConanPythonRequire, PyRequireLoader
+from conans.client.graph.range_resolver import RangeResolver
+from conans.client.hook_manager import HookManager
+from conans.client.loader import ConanFileLoader
+from conans.client.migrations import ClientMigrator
+from conans.cli.output import ConanOutput
+from conans.client.remote_manager import RemoteManager
+from conans.client.rest.auth_manager import ConanApiAuthManager
+from conans.client.rest.conan_requester import ConanRequester
+from conans.client.rest.rest_client import RestApiClientFactory
+from conans.client.runner import ConanRunner
+from conans.client.store.localdb import LocalDB
+from conans.client.tools.env import environment_append
+from conans.client.userio import UserIO
+from conans.errors import NoRemoteAvailable
+from conans.model.version import Version
+from conans.paths import get_conan_user_home
+from conans.tools import set_global_instances
+from conans.util.conan_v2_mode import CONAN_V2_MODE_ENVVAR
+from conans.util.env_reader import get_env
+from conans.util.files import exception_message_safe
+from conans.util.log import configure_logger
+from conans.util.tracer import log_command, log_exception
+from tqdm import tqdm
+
+
+def api_method(f):
+    def wrapper(api, *args, **kwargs):
+        quiet = kwargs.pop("quiet", False)
+        try:  # getcwd can fail if Conan runs on an unexisting folder
+            old_curdir = os.getcwd()
+        except EnvironmentError:
+            old_curdir = None
+        old_output = api.user_io.out
+        quiet_output = ConanOutput(StringIO(), color=api.color) if quiet else None
```

Comment:
IMHO, if we are using the `logging` library, the _quiet_ log should be implemented by removing the handlers

Reply:
I'll remove the quiet mode for the moment, I think it's not necessary for the next iteration of the cli...